tool/test-annocheck.sh

```diff
@@ -27,7 +27,8 @@ else
   # volume in container in container on GitHub Actions
   # <.github/workflows/compilers.yml>.
   TAG="${TAG}-copy"
-  "${DOCKER}" build --rm -t "${TAG}" --build-arg=FILES="${*}" -f ${TOOL_DIR}/annocheck/Dockerfile-copy .
+  sed -r "s/\\$\{FILES\}/${*}/" ${TOOL_DIR}/annocheck/Dockerfile-copy | \
```

Comment:
Isn't `/` as the separator unsafe, as it is also a file separator.

Reply:
@nobu I would agree with your opinion. the `/` is unsafe when the `${*}` includes the file separator `/`.

However, I don't think we need to use the `sed` command. Because this hack just ignores the functionality of the `--build-arg=FILES="${*}"` as an argument. And I found alternative ideas to deal with this limitation in a way to understand the logic easily in my opinion. I will share it soon.